packages/salesforcedx-vscode-lwc/src/index.ts

```diff
@@ -223,3 +226,11 @@ function startLWCLanguageServer(context: ExtensionContext) {
   // client can be deactivated on extension deactivation
   context.subscriptions.push(client);
 }
+
+export function getGlobalStore(): vscode.Memento {
```

Comment:
Let's move this to a separate class/util and not part of the main index. Same for `getWorkspaceSettings`